### ISSUE TITLE
‘About’ menu item is not getting activated via keyboard using Enter or Space key

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1461]
‘About’ menu item is not getting activated via keyboard using Enter or Space key

**Repro steps**
1. Launch 'Calculator'.
2. Home screen will be displayed.
3. Navigate to 'open Navigation' button and activate it.
4. Navigate to ‘About’ menu item and activate it.
5. Now verify that ‘About’ menu item is keyboard accessible or not

**Actual Result**
‘About’ menu item is not getting activated via keyboard using Enter or Space key.

**Expected Result**
‘About’ menu item in Navigation pane should be accessible via keyboard and get activated using 'Enter' key.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
